sitedata/config.yml

```diff
@@ -14,6 +14,7 @@ page_title:
   separator: ': '
 background_image: static/images/main.jpg
 organization: MiniConf Organization Committee
+chat_server: 35.232.236.127:3000
```

Comment:
Do you want to check in this specific IP address? Or maybe use a generic `localhost` for debugging purpose?

Reply:
Oh that's a working mini-conf test server. Maybe I'll get it a temp hostname.